MonitorizareVot/UIColor+Presets.swift

```diff
@@ -1,45 +1,20 @@
 //  Created by Code4Romania
 
-import Foundation
 import UIKit
 
 extension UIColor {
     
-    static let navigationBarBackground = UIColor(hexCode: 0xFFCC00)
-    static let navigationBarTint = UIColor(hexCode: 0x333E48)
+    static var colorSchema: ColorSchema.Type {
+        // UserInterfaceStyle is available from iOS 12, but dark mode is only available from iOS 13.
+        if #available(iOS 13.0, *) {
+            return UIApplication.shared.keyWindow?.rootViewController?.traitCollection.userInterfaceStyle == .light ?
```

Comment:
Use `UIScreen.main.traitCollection` instead of the root VC, since the traits are only applied to the topmost VC and the color mode isn't picked up. The screen gets the trait updates all the time.

Reply:
Makes sense, will update.

On the topic of colour changes being applied when trait collection changes, I would be a little weary of having to manually run an update of screen components when it changes, on every screen. That would be easy to get wrong and a big overhead to constantly carry on future development.

My thinking was to keep a version of the `userInterfaceStyle` inside the AppDelegate at launch and update it every time app comes into foreground. If it changes, refresh all views in memory. Problem is I haven't had time to play around with it and see exactly how to do that.

Reply:
Updated, but used instead `TraitCollection.current`. Having looked into it, that is more consistent if the code was to be run in an extension of the app, as well as enabling testing. We're able to override it in `TraitCollection.current`, but not within `UIScreen.main.traitCollection`.

If I get some time I'll look at forcing existing screens to redraw when app comes to background with a new interface style, but for now I think it would be safe to get this merged.

Reply:
When I put breakpoints in traitcollectiondidchange, they don't get triggered. Tried in simulator and my personal device running iOS 13, installed from Xcode.

Could you please confirm it works for you? From the docs, the method should be called, however I hope there's some project setting on my end that's causing this unusual issue for me.